### PR TITLE
ReaperX Presence

### PR DESCRIPTION
**Pull request by InwTanRew**
image
image
image
image

### CODE REVIEW

File: websites/R/ReaperX/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "id": "439582316873121793"
+    },
+    "service": "ReaperX",
+    "description": {
+      "en": "ReaperX is a R$ Shop Easy Fast Secure",
+      "th": "ReaperX ร้านรับเติม ROBUX (R$, โรบัค) ในเกม ROBLOX ที่ใช้ง่าย บริการจัดส่งอัตโนมัติ เรทสุดคุ้ม 5.5 - 6 เเบบกลุ่มเเละ ID - Pass เรท 4 - 9"
+    },
+    "url": [
+      "reaperx.in.th"
+    ],
```
Comment (Bas950) [inline websites/R/ReaperX/dist/metadata.json:13]: doesnt need to be in an array

File: websites/R/ReaperX/presence.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
+const presence = new Presence({
+    clientId: "751384892570468404"
+});
+  
+const browsingStamp = Math.floor(Date.now() / 1000);
+const path = document.location.pathname; //document.location.search;
```
Comment (Bas950) [inline websites/R/ReaperX/presence.ts:6]: merge consts and remove comment

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/R/ReaperX/dist/metadata.json (hunk 1 above)
Comment (InwTanRew) [reply websites/R/ReaperX/dist/metadata.json]: Here You Are. Fixed

**Review by doomlerd — DISMISSED** [review]

**Comment by InwTanRew** [comment]
sorry

File: websites/R/ReaperX/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    "service": "ReaperX",
+    "description": {
+      "en": "ReaperX is a R$ Shop Easy Fast Secure",
+      "th": "ReaperX ร้านรับเติม ROBUX (R$, โรบัค) ในเกม ROBLOX ที่ใช้ง่าย บริการจัดส่งอัตโนมัติ เรทสุดคุ้ม 5.5 - 6 เเบบกลุ่มเเละ ID - Pass เรท 4 - 9"
+    },
+    "url": "reaperx.in.th",
+    "version": "1.0.0",
+    "logo": "https://imgur.com/a/I97zZ4G",
+    "thumbnail": "https://imgur.com/a/ChK33dw",
```
Comment (Bas950) [inline websites/R/ReaperX/dist/metadata.json:14]: use the i.imgur links of the pictures

File: websites/R/ReaperX/presence.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({
+    clientId: "751384892570468404"
+});
+  
+const browsingStamp = Math.floor(Date.now() / 1000);
+const path = document.location.pathname;
```
Comment (Bas950) [inline websites/R/ReaperX/presence.ts:6]: combine consts

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/R/ReaperX/presence.ts (hunk 4 above)
Comment (InwTanRew) [reply websites/R/ReaperX/presence.ts:6]: sorry , What do you mean combine const am so confusing

Comment (Bas950) [reply websites/R/ReaperX/presence.ts:6]:
```
const var1 = val;
const var2 = val;
```

To:

```
const var1 = val,
          var2 = val;
```

Comment (InwTanRew) [reply websites/R/ReaperX/presence.ts]: Is it correct that i upload it?

**Review by Bas950 — DISMISSED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Pull request merged**